projects/plugins/jetpack/modules/likes/jetpack-likes-settings.php

```diff
@@ -196,22 +205,25 @@ function sharing_page() {
 			do_action( 'pre_admin_screen_sharing' );
 			?>
 			<?php $this->sharing_block(); ?>
-		</div> <?php
+		</div> 
+		<?php
 	}
 
 	/**
 	 * Returns the settings have been saved message.
 	 */
-	function updated_message() {
-		if ( isset( $_GET['update'] ) && $_GET['update'] == 'saved' ){
+	public function updated_message() {
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended -- ignoring nonce verification $_GET['update'] since we are strongly checking for its value. 
```

Comment:
This one seems fine to ignore, since we're only checking it to display a "settings have been saved" message. The worst an attacker could do would be to have that message displayed even though no settings were actually saved.

The reasoning given in the comment doesn't make sense to me though. It doesn't matter that we're "strongly checking for its value", what matters is that we're not making any change to the site in response to it.